package.json

```diff
@@ -8,6 +8,14 @@
   "main": "dist/chartjs-plugin-annotation.js",
   "module": "dist/chartjs-plugin-annotation.esm.js",
   "types": "types/index.d.ts",
+  "jsdelivr": "dist/chartjs-plugin-annotation.min.js",
+  "unpkg": "dist/chartjs-plugin-annotation.min.js",
+  "exports": {
+    "types": "./types/index.d.ts",
```

Comment:
small thing, we don't specify it like this anywhere in the package.json, in my oppinion we should keep in consistent, so either here we need to remove the `./` or add it everywhere else (has my preference)

Reply:
in exports, it needs to start with a dot where referencing a file. it could be a package too.

Reply:
Apoligize... I don't understand if it's correct.... Im CHART.,JS there is the dot:

https://github.com/chartjs/Chart.js/blob/b51c52336624a24fe1ec814dc47f9a4cb7dd6a8a/package.json#L16-L32